std/stdio.d

```diff
@@ -1448,11 +1448,22 @@ Throws: $(D Exception) if the file is not opened.
 
 /**
 Writes its arguments in text format to the file, according to the
-format in the first argument.
+format string fmt.
```

Comment:
This is not really to do with this PR, but it would be nice for the docs to reference `std.format` here so that the reader knows where to read up more on what a format string is.